airflow/contrib/operators/bigquery_operator.py

```diff
@@ -497,8 +482,8 @@ def execute(self, context):
             quote_character=self.quote_character,
             allow_quoted_newlines=self.allow_quoted_newlines,
             allow_jagged_rows=self.allow_jagged_rows,
-            src_fmt_configs=self.src_fmt_configs,
-            labels=self.labels
+            external_config_options=self.external_config_options,
```

Comment:
This will have to be for 2.0 as it will break things. Also, it needs to be backward-compatible to make updation smooth from 1.X to 2.0

Reply:
Haven't yet reviewed the entire PR but schemed through it. Will try to find some time to look at it more thoroughly